psalm-baseline.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The risky comparisons are easy to fix:

```php
if (is_iterable($options) || $options instanceof ExpectedClass) {
    // Do stuff
}

// Instead of:
if ($options) {}
```

Please can you address these instead of creating a new baseline.

Bear in mind that the expected Options classes may not be `iterable` so you should check for the correct type. i.e. `Laminas\Stdlib\AbstractOptions` does not implement `Traversable`